krux_boto/util.py

```diff
@@ -39,3 +40,29 @@ def get_instance_region():
         get_logger('krux_boto').warn('get_instance_region failed to get the local instance region')
         raise Error('get_instance_region failed to get the local instance region')
     return zone.rstrip(string.lowercase)
+
+
+# Region codes
+class RegionCode(Enum):
+    ASH = ('ASH', 'us-east-1')
+    PDX = ('PDX', 'us-west-2')
+    DUB = ('DUB', 'eu-west-1')
+    SIN = ('SIN', 'ap-southeast-1')
```

Comment:
You could create an `Enum` class that's used as keys for a `RegionCodes` class that extends `dict`. Alternatively, if we have non-normalized keys, some lookup logic could be put into `__getitem__`.